testcontainers-dapr/pom.xml

```diff
@@ -0,0 +1,135 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.dapr</groupId>
+        <artifactId>dapr-sdk-parent</artifactId>
+        <version>1.12.0-SNAPSHOT</version>
+    </parent>
+
+	<artifactId>testcontainers-dapr</artifactId>
+	<name>testcontainers-dapr</name>
+	<description>Testcontainers Dapr Module</description>
+	<packaging>jar</packaging>
+
+	<properties>
+		<snakeyaml.version>2.0</snakeyaml.version>
```

Comment:
I can definitely push this versions up.. I need to check if we really need this.

Reply:
good point.. we will just use the dapr-sdk

Reply:
done